boards/Makefile.common

```diff
@@ -108,7 +108,10 @@ all:	target/$(TARGET)/release/$(PLATFORM).bin
 lst:	target/$(TARGET)/release/$(PLATFORM).lst
 
 .PHONY: debug
-debug:	target/$(TARGET)/debug/$(PLATFORM).bin
+debug:  target/$(TARGET)/debug/$(PLATFORM).bin
```

Comment:
Inconsistent use of tabs vs space, I don't care whether tabs or spaces are used only that we are consistent!

Reply:
oh haha I thought you were pointing to a different line :). The other targets have a tab after the ":"! I just copied.